packages/card/src/css/index.js

```diff
@@ -104,6 +104,9 @@ export default {
     pointerEvents: 'none',
     opacity: 0
   },
+  '.psds-card__full-overlay:focus-within': {
```

Comment:
you also need to define the `[focus-within]` attr selector. that's what is added by the polyfill in unsupporting browsers